### DISCUSSION TITLE
execute on the UI from the boundobject

### DISCUSSION THREAD

**Discussion by shaharzi** [cefsharp/CefSharp#3749] [General]
i have a js call to the bound object, and in C# i have a method which gets URL from the JS client. im want to load this URL by calling the broswer.load(url), but nothing happens.  

js - BoundObject.load(url)

c# - `public void load(string url) {

            browser.Load(url);
        }`

**Comment by amaitland** [comment]
Please edit your original question and provide a more detailed example.

**Reply by amaitland** [reply]
Please also format your code so it's easy to read.

https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks
